tests/hpc/barrier_init.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Initialization of barriers for HPC multimachine tests
+# Maintainer: Petr Cervinka <pcervinka@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+use lockapi;
+use utils;
+
+sub run {
+    # Get number of nodes
+    my $nodes = get_required_var("CLUSTER_NODES");
+
+    # Initialize barriers
+    if (check_var("HPC", "slurm")) {
+        barrier_create("SLURM_MASTER_SERVICE_ENABLED", $nodes);
```

Comment:
You are not using barriers properly here - it works, but you can do the same with 1 or 2 barries, not 5.
For example, you should be able to replace `SLURM_MASTER_SERVICE_ENABLED` and `SLURM_SLAVE_SERVICE_ENABLED` with `SLURM_SERVICE_ENABLED`.

Reply:
Those are different points in tests, slurm must be ready on master first, than we need to wait for slaves.

Reply:
Then you are right, I did not know that